src/power_grid_model/validation/validation.py

```diff
@@ -221,6 +221,28 @@ def validate_ids_exist(update_data: Dict[str, np.ndarray], input_data: SingleDat
     return list(chain(*errors))
 
 
+def process_power_sigma_and_p_q_sigma(data: SingleDataset, sensor: str) -> None:
```

Comment:
this may actively make changes to the user input data set. that might not be desirable, right? why not just create a validation check?

Reply:
> you may in that case remove it from the standard list of required fields, and just make a custom check next to that check. In more formal terms: you make the fields optional and impose another sanity check, similar to how is done for the valid range of e.g. s0 for transformers
This of course can be done. I would argue for simplicity here. It was made clear to users that with such input combination, one is always ignored. By logic they shouldn't  expect that we leave that nan and not put a reasonable value in its place. Besides, this will also require changes to many test cases that explicitly check `assert (MissingValueError("sym_power_sensor", "power_sigma", [NaN]) in required_values_errors) == [something something]`

Reply:
I agree with @mgovers The input validation is just a validation. We should never change the input itself here.

Reply:
As a user, i own the data. Who knows if i attached some kind of database logic to any changes in my data. If i call a function that does not explicitly mention side effects, i do not expect any, as it may have extensive consequenses. It also makes usage of this functionality (validate and then investigate why it happened) difficult

Reply:
@mgovers @petersalemink95 Would the idea of doing a check on proxy appeal to you? That way the original input data will stay intact/untouched.

Reply:
@mgovers @petersalemink95 Input data is left untouched in the latest changes.

Reply:
If it works, then it's fine, but please add a test that proves that

Reply:
Changes pushed. Each power sensor got its individual validation w.r.t. `power_sigma`.